conf/views/controlDecision.njk

```diff
@@ -0,0 +1,69 @@
+{% extends "includes/layout.njk" %}
+
+{% from "macros/title.njk"                         import title %}
+
+{% block pageTitle %}
+  {{ title(messages("controlDecision.title")) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  <div class="govuk-main-wrapper">
```

Comment:
Do we need a back button? none present

Reply:
Great spot, added that in.

Reply:
Might need to investigate it more as it just adds the # unto the URL but it is the same for the other back links on manage frontend. So that can be done in another ticket.